run_tests.sh

```diff
@@ -2,4 +2,4 @@
 set -e
 SCRIPT_DIR=$(dirname `readlink -f $0`)
 $SCRIPT_DIR/manage.py test --noinput --settings=atmosphere.settings -v2 -- $@
-$SCRIPT_DIR/manage.py test --noinput --liveserver=localhost:8082 --settings=atmosphere.settings $SCRIPT_DIR/tests -- $@
+#$SCRIPT_DIR/manage.py test --noinput --liveserver=localhost:8082 --settings=atmosphere.settings $SCRIPT_DIR/tests -- $@
```

Comment:
are you keeping this commented-out line for post-Summit work or should it be deleted?